docs/content/latest/legal/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Legal information
+headerTitle: Legal information
+linkTitle: Legal
+description: Learn about Yugabyte licenses and third party software.
+image: /images/section_icons/index/quick_start.png
+headcontent: Learn about YugabyteDB licenses and third party software.
+type: page
+section: MISC
+menu:
+  latest:
+    identifier: legal
+    weight: 3000
+---
+
+## Licenses
+
+Source code in the [YugabyteDB repository](https://github.com/yugabyte/yugabyte-db/) is variously licensed under the Apache License 2.0 and the Polyform Free Trial License 1.0.0. A copy of each license can be found in the [licenses](https://github.com/yugabyte/yugabyte-db/tree/master/licenses) directory.
+
+The build produces two sets of binaries:
+
+- The entire database with all its features (including the enterprise ones) are licensed under the [Apache License 2.0](./apache-license).
+- The binaries that contain `-managed` in the artifact and help run a managed service are licensed under the [Polyform Free Trial License 1.0.0](./polyform-license).
```

Comment:
this link should point to https://github.com/yugabyte/yugabyte-db/blob/master/licenses/POLYFORM-FREE-TRIAL-LICENSE-1.0.0.txt

Reply:
done